### DISCUSSION TITLE
How to enable smooth scroll on react

### DISCUSSION THREAD

**Discussion by AbidHasanPiash** [STRINGLABITSOLUTIONS/Bhojon#185] [Q&A] [answered]
react

**Comment by montasim** [comment, marked as answer]
npm i -S react-scroll

import { Link, animateScroll as scroll } from "react-scroll";

<Link
    activeClass="active"
    to="section1"
    spy={true}
    smooth={true}
    offset={-70}
    duration={500}
>

scrollToTop = () => {
    scroll.scrollToTop(); 
};
